bugbug/tools/code_review.py

```diff
@@ -314,57 +316,69 @@ def get_all_inline_comments(
+                    date_created = transaction_comment["dateCreated"]
+                    comment_content = transaction_comment["content"]["raw"]
+
+                    diff_id = transaction["fields"]["diff"]["id"]
+                    filename = transaction["fields"]["path"]
+                    start_line = transaction["fields"]["line"]
+                    end_line = (
+                        transaction["fields"]["line"]
+                        + transaction["fields"]["length"]
+                        - 1
+                    )
+                    # Unfortunately, we do not have this information for a limitation
+                    # in Phabricator's API.
+                    on_removed_code = None
+
+                    # store the last modified date and if the comments have been marked as done
+                    date_modified = transaction_comment["dateModified"]
+                    is_done = transaction["fields"]["isDone"]
+
+                    # TODO: we could create an extended dataclass for this
+                    # instead of adding optional fields.
+                    comment = InlineComment(
+                        filename,
+                        start_line,
+                        end_line,
+                        comment_content,
+                        on_removed_code,
+                        comment_id,
+                        date_created,
+                        date_modified,
+                        is_done,
                     )
 
-                transaction_comment = transaction["comments"][0]
-                comment_id = transaction_comment["id"]
-                date_created = transaction_comment["dateCreated"]
-                comment_content = transaction_comment["content"]["raw"]
-
-                diff_id = transaction["fields"]["diff"]["id"]
-                filename = transaction["fields"]["path"]
-                start_line = transaction["fields"]["line"]
-                end_line = (
-                    transaction["fields"]["line"] + transaction["fields"]["length"] - 1
-                )
-                # Unfortunately, we do not have this information for a limitation
-                # in Phabricator's API.
-                on_removed_code = None
-
-                # TODO: we could create an extended dataclass for this
-                # instead of adding optional fields.
-                comment = InlineComment(
-                    filename,
-                    start_line,
-                    end_line,
-                    comment_content,
-                    on_removed_code,
-                    comment_id,
-                    date_created,
-                )
-
-                if not comment_filter(comment):
-                    continue
+                    if not comment_filter(comment):
+                        continue
 
-                diff_comments[diff_id].append(comment)
+                    diff_comments[diff_id].append(comment)
 
+            except Exception as e:
```

Comment:
We should avoid catching overly broad exception types whenever possible.

Reply:
I've removed it in b9803344f026f9040f1ac3fb1e4ba48f5394004d